### PR TITLE
Changes for Oryx pipeline migration.

### PR DESCRIPTION
**Pull request by SarathBussa**
Onboarding runtime and build images onto oneBranch

### CODE REVIEW

File: V2/build/__goVersions.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# This file was auto-generated from 'constants.yaml'. Changes may be overridden.
+
+GO_VERSION='1.15.8'
```
Comment (kumaraksh1) [inline V2/build/__goVersions.sh:3]: I think Sarthak has already updated go version. but let me confirm it.

File: images/build/Dockerfiles/cliBuilder.Dockerfile [hunk 2]
```diff
@@ -92,19 +118,17 @@ RUN apt-get update \
...
     && . ${BUILD_DIR}/__nodeVersions.sh \
-    && ${IMAGES_DIR}/receiveGpgKeys.sh 6A010C5166006599AA17F08146C2130DFD2497F5 \
-    && ${IMAGES_DIR}/retry.sh "curl -fsSLO --compressed https://yarnpkg.com/downloads/$YARN_VERSION/yarn-v$YARN_VERSION.tar.gz" \
-    && ${IMAGES_DIR}/retry.sh "curl -fsSLO --compressed https://yarnpkg.com/downloads/$YARN_VERSION/yarn-v$YARN_VERSION.tar.gz.asc" \
-    && gpg --batch --verify yarn-v$YARN_VERSION.tar.gz.asc yarn-v$YARN_VERSION.tar.gz \
     && mkdir -p /opt/yarn \
-    && tar -xzf yarn-v$YARN_VERSION.tar.gz -C /opt/yarn \
-    && mv /opt/yarn/yarn-v$YARN_VERSION /opt/yarn/$YARN_VERSION \
-    && rm yarn-v$YARN_VERSION.tar.gz.asc yarn-v$YARN_VERSION.tar.gz
+    && tar -xzf yarn-v1.22.15.tar.gz -C /opt/yarn \
```
Comment (kumaraksh1) [inline images/build/Dockerfiles/cliBuilder.Dockerfile:128]: why you preferred to hardcode the value?

File: images/runtime/commonbase/nodeRuntimeBase.Dockerfile [hunk 3]
```diff
@@ -15,13 +15,10 @@ RUN apt-get update \
...
-RUN curl -fsSLO --compressed "https://yarnpkg.com/downloads/$YARN_VERSION/yarn-v$YARN_VERSION.tar.gz" \
-  && curl -fsSLO --compressed "https://yarnpkg.com/downloads/$YARN_VERSION/yarn-v$YARN_VERSION.tar.gz.asc" \
-  && gpg --batch --verify yarn-v$YARN_VERSION.tar.gz.asc yarn-v$YARN_VERSION.tar.gz \
-  && mkdir -p /opt \
-  && tar -xzf yarn-v$YARN_VERSION.tar.gz -C /opt/ \
-  && ln -s /opt/yarn-v$YARN_VERSION/bin/yarn /usr/local/bin/yarn \
-  && ln -s /opt/yarn-v$YARN_VERSION/bin/yarnpkg /usr/local/bin/yarnpkg \
-  && rm yarn-v$YARN_VERSION.tar.gz.asc yarn-v$YARN_VERSION.tar.gz
+RUN mkdir -p /opt \
+  && tar -xzf yarn-v1.22.15.tar.gz -C /opt/ \
```
Comment (kumaraksh1) [inline images/runtime/commonbase/nodeRuntimeBase.Dockerfile:21]: same here

File: images/build/Dockerfiles/azureFunctions.JamStack.Dockerfile [hunk 4]
```diff
@@ -73,22 +80,16 @@ RUN set -ex \
...
-RUN --mount=type=secret,id=oryx_sdk_storage_account_access_token \
-    set -e \
-    && export ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN_PATH="/run/secrets/oryx_sdk_storage_account_access_token" \
+COPY images/yarn-v1.22.15.tar.gz .
+RUN set -e \
     && yarnCacheFolder="/usr/local/share/yarn-cache" \
     && mkdir -p $yarnCacheFolder \
     && chmod 777 $yarnCacheFolder \
-    && . ${BUILD_DIR}/__nodeVersions.sh \
-    && if [ "${DEBIAN_FLAVOR}" == "bullseye" || "${DEBIAN_FLAVOR}" == "buster" ]; then ${IMAGES_DIR}/installPlatform.sh nodejs ${NODE16_VERSION}; fi \
```
Comment (sarsharma) [inline images/build/Dockerfiles/azureFunctions.JamStack.Dockerfile:83]: looks like we are still missing installing node here

File: images/build/Dockerfiles/cli.Dockerfile [hunk 5]
```diff
@@ -3,7 +3,7 @@ ARG DEBIAN_FLAVOR
 # Use the curl flavor of buildpack-deps as the base image, which is lighter than the standard flavor; more information here: https://hub.docker.com/_/buildpack-deps
 FROM buildpack-deps:${DEBIAN_FLAVOR}-curl as main
 ARG DEBIAN_FLAVOR
-ARG SDK_STORAGE_BASE_URL_VALUE="https://oryx-cdn.microsoft.io"
```
Comment (sarsharma) [inline images/build/Dockerfiles/cli.Dockerfile:6]: should we revert this?

File: images/runtime/python/template.Dockerfile
Comment (sarsharma) [inline images/runtime/python/template.Dockerfile:1]: are we using template docker file for python? looks like we might be missing some files here due to gitignore

File: platforms/php/composer-setup.php
Comment (sarsharma) [inline platforms/php/composer-setup.php:1]: was this checked in by mistake?

File: tests/Oryx.BuildImage.Tests/CommandTests/PlatformsCommandTest.cs [hunk 6]
```diff
@@ -85,25 +85,7 @@ public void ListsPlatformsAndVersionsAvailableForDynamicInstall()
...
                     Assert.True(phpPlatform.Versions.Any());
-                    Assert.True(phpPlatform.Versions.Contains("8.1.6"));
-
-                    var hugoPlatform = actualResults
-                        .Where(pr => pr.Name.EqualsIgnoreCase(HugoConstants.PlatformName))
-                        .FirstOrDefault();
-                    Assert.NotNull(hugoPlatform);
-                    Assert.NotNull(hugoPlatform.Versions);
-                    Assert.True(hugoPlatform.Versions.Any());
-                    Assert.True(hugoPlatform.Versions.Contains(HugoConstants.Version));
```
Comment (sarsharma) [inline tests/Oryx.BuildImage.Tests/CommandTests/PlatformsCommandTest.cs:96]: let's not remove the test for Hugo

**Review by sarsharma — DISMISSED** [review]

**Review by sarsharma — APPROVED** [review]

**Review by kumaraksh1 — APPROVED** [review]
LGTM!

**Pull request merged**